source/manual/deploy-puppet.html.md

```diff
@@ -20,13 +20,19 @@ deploy.
 
 You can deploy Puppet to Staging and Production using the following steps:
 
-1. Deploy a newer version of govuk-puppet to staging by using the 'Deploy to Staging (Carrenza)'
-   and 'Deploy to Staging (AWS)' buttons in the Release app after clicking on the release
-   tag and reviewing the changes listed.
-1. You will either need to wait 30 minutes or read about [convergence](#convergence).
-   You should monitor Icinga and Smokey, and test anything you're concerned about.
+1. In the [Release app][release-app-puppet], click on the release tag and review
+   the changes listed.
+1. If your change on integration appears as expected, deploy a newer version of 
+   govuk-puppet to staging by using the 'Deploy to Staging' button.
+1. Wait 30 minutes or read about [convergence](#convergence).
+   You should monitor [Staging Icinga][icinga-staging] and [Staging Smokey][smokey-staging],
+   and test anything you're concerned about.
 1. Repeat the above steps for Production.
```

Comment:
I know this ins't part of your changes, but I don't think you need line 30. Line 21 already makes it clear that the following steps are for both environments.

Reply:
If we removed line 30 we should change some wording and possibly add extra points for production because all these steps (including the links) are specific to staging.

Reply:
ok no problem, leave it as is